File: LG_FFmpeg/app/src/main/java/com/louis/lg_ffmpeg/MainActivity.java
package com.louis.lg_ffmpeg;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import com.louis.lg_ffmpeg.databinding.ActivityMainBinding;
import com.louisgeek.ffmpeg.FFmpegUtil;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        // Example of a call to a native method
        TextView tv = binding.sampleText;
        tv.setText("FFmpeg 版本：" + FFmpegUtil.getFFmpegVersion());
    }

}

File: MyJni/app/src/main/java/com/louis/myjni/MainActivity.kt
package com.louis.myjni

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import com.louis.myjni.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        // Example of a call to a native method
        binding.sampleText.text = stringFromJNI()
    }

    /**
     * A native method that is implemented by the 'myjni' native library,
     * which is packaged with this application.
     */
    external fun stringFromJNI(): String

    companion object {
        // Used to load the 'myjni' library on application startup.
        init {
            System.loadLibrary("myjni")
        }
    }
}